vispy/visuals/graphs/layouts/circular.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+"""
+Circular Layout
+===============
+
+This module contains several graph layouts which rely heavily on circles.
+"""
+
+import numpy as np
+
+from ..util import straight_line_vertices
+
+
+def circular(adjacency_mat, directed=False):
+    """Places all nodes on a single circle.
+
+    Parameters
+    ----------
+    adjacency_mat : matrix
+        The graph adjacency matrix
```

Comment:
missing `directed` param

If `directed is False`, you should say what part of the matrix is used. Just upper-right, lower-left, or the union of the two?

Eventually we might want to allow the adjacency matrix to actually be weights instead of zeros and ones, too, BTW.